examples/quickstarts/cakephp-mysql-persistent.json

```diff
@@ -60,7 +60,10 @@
       "kind": "Route",
       "apiVersion": "v1",
       "metadata": {
-        "name": "${NAME}"
+        "name": "${NAME}",
+        "annotations": {
+          "template.openshift.io/expose": ""
+        }
```

Comment:
I clearly need to find a way to make this readme more visible :(

https://github.com/openshift/origin/blob/master/examples/quickstarts/README.md
```
Files in this directory are automatically pulled down, do not modify/add files to this directory.
```